src/tr1/game/lara/cheat.c

```diff
@@ -114,7 +114,7 @@ void Lara_Cheat_Control(void)
                 g_Lara.uzis.ammo = 5000;
                 Sound_Effect(SFX_LARA_HOLSTER, NULL, SPM_ALWAYS);
             } else if (as == LS_SWAN_DIVE) {
-                Effect_ExplodingDeath(g_Lara.item_num, -1, 0);
+                Effect_ExplodingDeath(g_Lara.item_num, -1, 1);
```

Comment:
This causes explosions similar to `/abortion` if the original TR2+ exploding Lara cheat is performed.

I did not change the `/kill` command to have explosions / damage since that is a testing cheat to kill an enemy and avoid enemies. I don't think it makes sense to cause explosions that hurt Lara in that scenario.

Reply:
Sounds good. Let's make sure we document this in the changelog, though.